unit-tests/src/test/scala/tests/SuiteSuite.scala

```diff
@@ -66,7 +66,9 @@ object SuiteSuite extends Suite {
         }
 
       case exc: Throwable =>
-        if (exc == AssertionFailed) // got case object, expected case class
+        if (exc.isInstanceOf[
+              AssertionFailed
+            ]) // got case object, expected case class
```

Comment:
scalafmt does something horrible here. Consider moving the comment above the if or inside the branch so that this doesn't happen.